### DISCUSSION TITLE
自定义鼠标手势触发按键（可以选择使用侧键拖动触发鼠标手势）

### DISCUSSION THREAD

**Discussion by enximi** [shuax/MouseInc.Settings#297] [Ideas]
现在鼠标手势只能通过右键触发的。我想用侧键触发鼠标手势。

在一些软件中右键拖动都是有相应操作的，例如各种建模软件，思维导图，一些网页中。想正常使用这些软件，就要一个个添加到禁用名单，比较麻烦。尤其是网页的情景中，我即想在浏览器中使用鼠标手势，又想正常使用网页原本的功能，就只能暂时禁用mouseinc。

而一般场景下，鼠标侧键除了点击触发的前进后退，几乎没有任何功能。就算有，也是通过鼠标宏或者编程自定义的，只在特定软件中起作用。

使用侧键触发鼠标手势可以避免上面那些冲突，在正常使用软件的同时也可以使用鼠标手势。

**Comment by shuax** [comment]
下一代会考虑，目前版本不做了。
